operators/registry-operator/0.1.2/manifests/registry-operator.clusterserviceversion.yaml

```
@@ -0,0 +1,860 @@
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - registry.devfile.io
+          resources:
+          - devfileregistrieslists/finalizers
+          verbs:
+          - update
+        - apiGroups:
+          - registry.devfile.io
+          resources:
+          - devfileregistrieslists/status
+          verbs:
+          - get
+          - patch
+          - update
+        - apiGroups:
+          - route.openshift.io
+          resources:
+          - routes
+          - routes/custom-host
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - authentication.k8s.io
+          resources:
+          - tokenreviews
+          verbs:
+          - create
+        - apiGroups:
+          - authorization.k8s.io
+          resources:
+          - subjectaccessreviews
+          verbs:
+          - create
+        serviceAccountName: registry-operator-service-account
+      deployments:
+      - label:
+          app.kubernetes.io/name: devfileregistry-operator
+          control-plane: controller-manager
+        name: registry-operator-controller-manager
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              app.kubernetes.io/name: devfileregistry-operator
+              control-plane: controller-manager
+          strategy: {}
+          template:
+            metadata:
+              annotations:
+                kubectl.kubernetes.io/default-container: manager
+              labels:
+                app.kubernetes.io/name: devfileregistry-operator
+                control-plane: controller-manager
+            spec:
+              containers:
+              - args:
+                - --secure-listen-address=0.0.0.0:8443
+                - --upstream=http://127.0.0.1:8080/
+                - --logtostderr=true
+                - --v=10
+                image: gcr.io/kubebuilder/kube-rbac-proxy:v0.13.1
+                name: kube-rbac-proxy
+                ports:
+                - containerPort: 8443
+                  name: https
+                  protocol: TCP
+                resources:
+                  limits:
+                    cpu: 500m
+                    memory: 128Mi
+                  requests:
+                    cpu: 5m
+                    memory: 64Mi
+                securityContext:
+                  allowPrivilegeEscalation: false
+                  capabilities:
+                    drop:
+                    - ALL
+                  seccompProfile:
+                    type: RuntimeDefault
+              - args:
+                - --health-probe-bind-address=:8081
+                - --metrics-bind-address=127.0.0.1:8080
+                - --leader-elect
+                command:
+                - /manager
+                image: quay.io/devfile/registry-operator:next
```

The current kiwi test failure is caused by this line, the tag should be the version tag rather than `next`, `v0.1.2` for example should apply the following change:
```suggestion
                image: quay.io/devfile/registry-operator:v0.1.2
```